numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -95,6 +97,76 @@ def test_linking_lazy_compile(self):
     def test_linking_eager_compile(self):
         self._test_linking(eager=True)
 
+    @skip_unless_cuda_python('NVIDIA Binding needed for NVRTC')
+    def test_linking_cu(self):
+        bar = cuda.declare_device('bar', 'int32(int32)')
+
+        link = os.path.join(os.path.dirname(__file__), 'data', 'jitlink.cu')
```

Comment:
Is `jitlink.cu` missing from this patch?

Reply:
It already exists: https://github.com/numba/numba/blob/main/numba/cuda/tests/cudadrv/data/jitlink.cu

Reply:
ah yes, thanks.